src/libraries/System.Runtime.InteropServices/tests/ComDisabled/System.Runtime.InteropServices.ComDisabled.Tests.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppCurrent)-windows</TargetFrameworks>
+    <TestRuntime>true</TestRuntime>
+  </PropertyGroup>
```

Comment:
Are all the tests in this test suite Window-only? If so, we could set `IgnoreForCI` to true for non-Windows with a comment as to why (since it would change if we do cross-plat ComWrappers), so that we avoid making the CI do unnecessary work.